langchain/evaluation/run_evaluators/string_run_evaluator.py

```diff
@@ -383,3 +384,71 @@ def from_model_and_evaluator(
             example_mapper=example_mapper,
             string_evaluator=evaluator,
         )
+
+    @classmethod
+    def from_run_and_data_type(
+        cls,
+        evaluator: StringEvaluator,
+        run_type: RunTypeEnum,
```

Comment:
This method isn't meant to really be called outside runner utils and perhaps is confusing since you have to think about runner type  and data type